src/app/base.jsx

```diff
@@ -0,0 +1,28 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+
+/**
+ * Base html document component.
+ * Served as static html, wraps single-page app.
+ */
+
+module.exports = React.createClass({
+  render: function() {
+    return (
+      <html>
+        <head>
+          <title>{this.props.title}</title>
```

Comment:
A component's "render" function takes this.state and this.props and does a transformation. It ingests json and produces a virtual dom representation:
http://facebook.github.io/react/docs/component-specs.html